DevOps/README.md

```diff
@@ -1,5 +1,34 @@
 ## DEVOPS
 
+## Version Control System - Git & Github
+
+Automate, customize, and execute your software development workflows right in your repository with GitHub Actions. You can discover, create, and share actions to perform any job you'd like, including CI/CD, and combine actions in a completely customized workflow.
+
+Visit the following resources to learn more:
+
+- [GitHub Awesome Explanation By Kunal Kushwaha ](shorturl.at/bFIT1)
+- [GitHub Actions Documentation](https://docs.github.com/en/actions)
+- [Learn GitHub Actions](https://docs.github.com/en/actions/learn-github-actions)
+- [GitHub Actions - Supercharge your GitHub Flow](https://youtu.be/cP0I9w2coGU)
+- [Automate your Workflow with GitHub Actions](https://www.youtube.com/watch?v=nyKZTKQS_EQ)
```

Comment:
Please keep it the exact name of the resource. The resource name is not `GitHub Awesome Explanation` :)